backend/src/v5/processors/teamspaces/projects/models/drawings.js

```diff
@@ -94,7 +97,50 @@ Drawings.downloadRevisionFiles = async (teamspace, drawing, revision) => {
 		throw templates.fileNotFound;
 	}
 
-	return getFileAsStream(teamspace, `${DRAWINGS_HISTORY_COL}.ref`, rev.rFile[0]);
+	return getFileAsStream(teamspace, DRAWINGS_HISTORY_COL, rev.rFile[0]);
+};
+
+Drawings.getLatestThumbnail = async (teamspace, project, drawing) => {
+	try {
+		const rev = await getLatestRevision(teamspace, drawing, modelTypes.DRAWING, { thumbnail: 1 });
+
+		if (!rev?.thumbnail) {
```

Comment:
would rather we stick with this approach since it's virtually the same code

But to do it like you suggested we're making the assumption that it'll always throw, or it'll always be an object. Which whilst with the current assumption it stands, you never know and we're just giving ourselves more opportunity to run into errors.